README.md

```diff
@@ -17,6 +17,34 @@ As an input param it takes a json string in directus filter format and applies t
 
 ```
 
+## Queryable usage
+
+When you implement the IApiClient interface into your API client class, you are able to run linq queries
+against it like this:
+
+``` csharp
+using RuleFilterParser;
+
+IApiClient client = ...;
+
+var queryable = client.GetAsQueryable("persons");  // URL path of the API
```

Comment:
How is the type of "persons" inferred in this example?

Reply:
Hi Reng, the Readme is outdated, I will fix it.

Take a look at the [ApiClientMockup](https://github.com/bcc-code/rule-filter-parser-dotnet/pull/23/files#diff-9e95210a838951dd95e96d38985089a8c4cccd2734ea58765daba544e8b78631R6-R16) and the [Unit tests in LinqTests.cs](https://github.com/bcc-code/rule-filter-parser-dotnet/pull/23/files#diff-59125ee4a64c41e8b3cca41a435bd6c8879a5b8e4912a9c651f0313b38a5cad4R8-R21) to see how a possible API client is implemented.

I think it would be great to discuss the implementation today at WWW. I am open for changes. Especially the implementation for async. linq calls needs some rethinking ... but maybe that could be a new PR after that one here.

Reply:
I agree that a new PR would be good. I propose just approving this PR and then working in smaller batches. Also testing using this in the Core API SDK to see if there are any gaps or things that are substantially different compared to the previous solution.

Reply:
It might be nice to standardize on the BccCode namespace prefix. E.g. BccCode.Library or something like that

Reply:
I agree. I've merged the PR now.